Gemfile.lock

```diff
@@ -266,125 +268,127 @@ GEM
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
+    bigdecimal (3.1.6)
     binding_of_caller (0.8.0)
       debug_inspector (>= 0.0.1)
-    bootsnap (1.4.5)
-      msgpack (~> 1.0)
+    bootsnap (1.18.3)
+      msgpack (~> 1.2)
     bootstrap-sass (3.4.1)
       autoprefixer-rails (>= 5.2.1)
       sassc (>= 2.0.0)
     bootstrap3-datetimepicker-rails (4.17.47)
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
```

Comment:
### [dogstatsd-ruby](https://github.com/DataDog/dogstatsd-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 3.0.0 → 3.3.0

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/dogstatsd-ruby/3.0.0/3.3.0)_]





<details>
<summary>Commits</summary>

A change of **59** commits. See the full changes on [the compare page](https://github.com/DataDog/dogstatsd-ruby/compare/v3.0.0...v3.3.0).

These are the first 10 commits:
- (31e5494) [Adds failing spec for symbol tag value](https://github.com/DataDog/dogstatsd-ruby/commit/31e54941dd240ccf6e9353879835eec422a097ae)
- (3bc17a5) [`to_s` all the tags](https://github.com/DataDog/dogstatsd-ruby/commit/3bc17a54ee53b5ad53d4ae677f1586cbd37161e5)
- (7f8d8b7) [Ignore nil tags and convert symbol ones](https://github.com/DataDog/dogstatsd-ruby/commit/7f8d8b740f071766956a8c844e183a1e787e45c3)
- (0bf5236) [Add unix socket support](https://github.com/DataDog/dogstatsd-ruby/commit/0bf5236ea448a934b24d71f49946f5d1863eb35b)
- (c64b371) [Use nonblocking send for unix socket](https://github.com/DataDog/dogstatsd-ruby/commit/c64b371f8e8cd6bbcb8362c2e113ae51ef91888d)
- (1ee2a5f) [Use defaults when host/port are nil](https://github.com/DataDog/dogstatsd-ruby/commit/1ee2a5f4fd3b02824b521253d9be081868170e0f)
- (8b816f8) [Don't send anything when the buffer is empty](https://github.com/DataDog/dogstatsd-ruby/commit/8b816f89748cd1365fe542610b2a7153822e612d)
- (bdf6065) [Merge pull request #53 from pschambacher/pschambacher/backwa…](https://github.com/DataDog/dogstatsd-ruby/commit/bdf606578e24da642548ffd6f3a93f471a5274e6)
- (30d0e0e) [Merge pull request #51 from jacobbednarz/to-s-all-the-tags](https://github.com/DataDog/dogstatsd-ruby/commit/30d0e0e5932715084d0cc87bf5de335335472592)
- (518fa60) [Fix error handling when using Unix Socket](https://github.com/DataDog/dogstatsd-ruby/commit/518fa60b437ff561a54cc0b3f96f26d0b1029467)


</details>